<launch>
    <!-- Parameters -->
    <arg name="setup_file"  default="rovio_euroc_mono_imu.launch" />
    <arg name="pose_topic"  default="/rovio/pose_with_covariance_stamped" />
    <arg name="msg_type"     default="PoseWithCovarianceStamped" />
    <arg name="output_file" default="$(find rudpt_evaluation)/eval_data/test.txt" />
    
    <!-- Setup the rovio environment with the given setup file -->
    <!-- <include file="$(find rudpt_evaluation)/launch/rovio/$(arg setup_file)" /> -->
    <include file="$(find rovio)/launch/rovio_node.launch" />
    
    <!-- Initialise node for saving the rovio poses -->
    <node name="path_writer_node" pkg="rudpt_evaluation" type="path_writer_node.py" output="screen" clear_params="true">
        <param name="pose_topic" value="$(arg pose_topic)" type="str" /> 
         <param name="msg_type" value="$(arg msg_type)" type="str" />
        <param name="output_file" value="$(arg output_file)" type="str" />
    </node>

</launch>

